mlflow/store/model_registry/file_store.py

```diff
@@ -942,7 +942,7 @@ def delete_model_version_tag(self, name, version, key):
 
     def _get_registered_model_alias_path(self, name, alias):
         _validate_model_name(name)
-        _validate_model_alias_name(alias)
+        _validate_model_alias_name(alias, perform_latest_check=False)
```

Comment:
From offline discussion, we don't want to remove validation from the OSS storage backends (file/SQL store)